Changelog.md

```diff
@@ -1,5 +1,13 @@
 ## Gazebo Rendering
 
+### Gazebo Rendering 8.2.2 (2025-01-30)
+
+1. Fix RenderingIface_TEST for ogre2 when there is no display
+    * [Pull request #1098](https://github.com/gazebosim/gz-rendering/pull/1098)
+
+1. Better approach to use std::variant
+    * [Pull request #1093](https://github.com/gazebosim/gz-rendering/pull/1093)
+
```

Comment:
move this block below `### Gazebo Rendering 8.X`?

Reply:
https://github.com/gazebosim/gz-rendering/pull/1105/commits/74a422bb69b0f0c67accb54f915e1511e6556b4c